src/components/Pane/Pane.Props.ts

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { Pane } from './Pane';
+
+export interface IPaneProps extends React.Props<Pane> {
+  /**
+   * Whether the pane is displayed.
+   * @default false
+   */
+  isOpen?: boolean;
+
+  /**
+   * Optional parameter stating whether the content box should squish (true) or scroll (false)
+   */
+  isOverlay?: boolean;
+
+  /**
+   * Has the close button visible.
+   * @default true
+   */
+  hasCloseButton?: boolean;
```

Comment:
Thanks, I did read your best practices update.  Trying to implement starting now.